src/test/java/nl/tudelft/contextproject/model/EntityTest.java

```diff
@@ -47,4 +55,42 @@ public void testSetState() {
 		entity.setState(EntityState.ALIVE);
 		assertEquals(entity.getState(), EntityState.ALIVE);
 	}
+	
+	/**
+	 * Test if collision detection returns true when it should.
+	 */
+	@Test
+	public void testPlayerCollisionTrue() {
+		TestUtil.mockGame();

```

Comment:
Should preferrably be done in an "@BeforeClass" method.
Take a look at WebTestBase for an example of a safe set/unset method.